docs/background/index.md

```diff
@@ -0,0 +1,7 @@
+# Backgroud
+
+This section contains discussions regarding various features and
+modes of operation {{brand}} provides. Those discussions are not
+concerned with how to do specific tasks, but rather with what
+features to take advantage of or how to go about achieving a
+set goal. 
```

Comment:
Just checking: any particular reason why you didn't just rename `docs/concepts/index.md` to `docs/background/index.md`?

Reply:
Uhm, I remember renaming the whole folder and then making some changes in some files :|

Reply:
OK, please do

```bash
git rm --cached docs/concepts/index.md
git commit --amend --no-edit
git push --force-with-lease
```